client/src/views/UserProfile.vue

```diff
@@ -5,13 +5,33 @@
       <!-- Profile Picture -->
       <div class="col-md-3">
         <div class="d-flex flex-column align-items-center text-center p-3">
-          <img
-            class="rounded-circle"
-            width="150px"
-            src="https://avatars.dicebear.com/api/big-ears/your-custom-seed.svg?b=%23d9eb37"
+          <img class="rounded-circle" :src="profileImageUrl" />
+          <span class="font-weight-bold">{{ userData.firstName }}</span>
+          <span class="text-secondary">{{ userData.lastName }}</span>
+          <div class="row gap-3">
+            <button
```

Comment:
Die Buttons finde ich etwas überpräsent. Ich denke nur die Symbole ohne Button drumherum sind eleganter und tragen nicht so auf.
Dann am besten noch oben rechts in den maskierten Teil des Bildes, oder zumindest direkt unter das Bild, sonst denkt man man kann die Unterschrift bearbeiten.

Reply:
done